pages/glossary/owner-element.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Owned by
+key: owned-by
+---
+
+An element A is owned by element B if both elements exist within the same [document tree](https://www.w3.org/TR/dom41/#document-trees) or [shadow tree](https://www.w3.org/TR/dom41/#shadow-trees) and either:
```

Comment:
This is incorrect. It suggests that elements in the light DOM tree can't own elements in a shadow DOM tree and vice versa. I'm not sure how to solve this though, the concept of flat tree  is not included in the DOM 4 spec.

Reply:
Never mind the DOM 4 spec. Flat tree is defined here: https://drafts.csswg.org/css-scoping/#flat-tree

Reply:
Agreed. However, while implicit ownership indeed can cross shadow borders, explicit ownership (`aria-owns`) cannot.

Reply:
@WilcoFiers and @kasperisager: The definition of "owned by" has been updated to consider this, and 2 test cases have been added to cover this: Passed Example 6 and Failed Example 7.